app/views/home/authenticated/index.erb

```diff
@@ -0,0 +1,20 @@
+<div class="usa-alert usa-alert--info">
+  <div class="usa-alert__body">
+    <p class="usa-alert__text">You can now create teams.</p>
```

Comment:
So, can't only government users create teams? Unless I'm mistaken it looks like everyone will see this message telling them they can.

Reply:
Thanks, I foolishly missed this 😬